### ISSUE TITLE
how to output the summary information every step ?

### ISSUE THREAD

**Issue by dongzhuoyao** [ppwwyyxx/tensorpack#509]
in case that I want debug whether the loss is descending.

**Comment by ppwwyyxx** [comment]
There are other ways to print tensors other than summaries: http://tensorpack.readthedocs.io/en/latest/tutorial/faq.html#how-to-print-dump-intermediate-results-in-training

Summaries are added to tensorboard in the callbacks:
http://tensorpack.readthedocs.io/en/latest/tutorial/summary.html#tensorflow-summaries .
And you can use a different `period` option: http://tensorpack.readthedocs.io/en/latest/modules/callbacks.html#tensorpack.callbacks.MergeAllSummaries
